src/Components/FavPage/FavPage.jsx

```diff
@@ -1,9 +1,27 @@
+import { useSelector, useDispatch } from 'react-redux';
+import { FavCitiesList, Message } from '../';
+
 import styles from './FavPage.module.scss';
 
 function FavPage() {
+  const favCityList = useSelector((state) => state);
+
+  const listItems = favCityList.map((item) => {
+    return <FavCitiesList data={item} key={item.locationData.id} />;
+  });
+
   return (
     <main className={styles.container}>
-      <h2>Favourite Locations</h2>
+      {listItems.length !== 0 && <h2 className={styles.title}>Favourite Locations</h2>}
+
+      {listItems.length !== 0 ? (
+        <div className={styles['container-items']}>{listItems}</div>
```

Comment:
why not combine two conditions in one like this?
if (listItems.length !== 0) {
  < title >
  < items >
} else {
  < message > 
}

Reply:
ok, changes made.